pyhoma/client.py

```diff
@@ -126,6 +126,19 @@ async def get_gateways(self, refresh: bool = False) -> List[Gateway]:
 
         return gateways
 
+    @backoff.on_exception(
+        backoff.expo, NotAuthenticatedException, max_tries=2, on_backoff=relogin
+    )
+    async def get_device_definition(self, deviceurl: str) -> JSON:
```

Comment:
Is `JSON` a valid return type? We use Dict for all others, but I am not sure why. Do you know @tetienne ?

Reply:
JSON is defined as 
```py
JSON = Union[Dict[str, Any], List[Dict[str, Any]]]
```

Reply:
It just makes it clearer IMO

Reply:
The guy who wrote that must be really nice :D

Reply:
Maybe we should have a look at other definitions as well! Good catch.